chainer/functions/connection/linear.py

```diff
@@ -180,13 +180,13 @@ def backward(self, x, gy):
         return (gx[0], func.gW, func.gb)
 
 
-def linear(x, W, b=None, stride=1, pad=0, use_cudnn=True):
+def linear(x, W, b=None):
     """Nonparameterized linear function.
 
     Args:
         x (~chainer.Variable): Input variable.
         W (~chainer.Variable): Weight variable.
-        b (~chainer.Variable): Bias variable.
+        b (~chainer.Variable): Bias variable (optional).
```

Comment:
Do we need to fix docstring of convolution2d like this?: https://github.com/pfnet/chainer/pull/369/files#diff-34a15eecaef34d920b685d1f971f6a17L371